x/nft/codec.proto

```diff
@@ -32,10 +32,23 @@ message AddApprovalMsg {
     bytes address = 2;
     string action = 3;
     ApprovalOptions options = 4 [(gogoproto.nullable) = false];
+    Type t = 5;
 }
 
 message RemoveApprovalMsg {
     bytes id = 1;
     bytes address = 2;
     string action = 3;
+    Type t = 4;
```

Comment:
pp: how about renaming this to `nftType`? Type and variable

Reply:
when we have a package name of nft, the nft prefix for the variable seems redundant